### PR TITLE
[devtools-translate] Spanish - What's New in DevTools (Chrome 96)

### PR DESCRIPTION
**Pull request by Caballerog**
[devtools-translate] Spanish - What's New in DevTools (Chrome 96)

### CODE REVIEW

**Comment by chybie** [comment]
Thanks @Caballerog , over to @midudev and @and-oli  for review!

File: site/es/blog/new-in-devtools-96/index.md [hunk 1]
```diff
@@ -28,171 +28,172 @@ draft: true
 
 {% include 'partials/devtools/es/banner.md' %}
 
-<!-- ## Preview feature: New CSS Overview panel {: #css-overview } -->
+## Función de vista previa: Nuevo panel de Descripción General de CSS {: #css-overview }
 
-<!-- Use the new **CSS Overview** panel to identify potential CSS improvements on your page.
-[Open the **CSS Overview** panel](/docs/devtools/css-overview#open), then click on **Capture overview** to generate a report of your page’s CSS. -->
+Utilice el nuevo panel **Descripción General de CSS** para identificar posibles mejoras de CSS en su página.
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-96/index.md:33]: It seems that we have different translations for CSS Overview in  "Spanish" and in "Spanish (Latin America)". This is unfortunate and should be fixed along with other inconsistencies with the newly added translations. However, in the mean time, could we please include the other translation ("Resumen de CSS") here as well?
```suggestion
Utilice el nuevo panel **Descripción General de CSS / Resumen de CSS** para identificar posibles mejoras de CSS en su página.
```

**Review by and-oli — COMMENTED** [review]
Thank you very much for the translation!

File: site/es/blog/new-in-devtools-96/index.md [hunk 2]
```diff
@@ -28,171 +28,172 @@ draft: true
...
 
-<!-- You can further drill down on the information. For example, click on a color in the **Colors** section to view the list of elements that apply the same color. Click on an element to open the element in the **Elements** panel. -->
+Puede profundizar más en la información. Por ejemplo, haga clic en un color en la sección **Colores** para ver la lista de elementos que aplican el mismo color. Haga clic en un elemento para abrir el elemento en el panel **Elementos**.
 
-<!-- The **CSS Overview** panel is a preview feature. Our team is still actively working on it and we are looking for your [feedback](https://goo.gle/css-overview-feedback) for further enhancements. -->
+El panel **Descripción General de CSS** es una función de vista previa. Nuestro equipo todavía está trabajando activamente en él y estamos buscando sus [comentarios](https://goo.gle/css-overview-feedback) para obtener más mejoras.
 
-<!-- Read [this article](/docs/devtools/css-overview) to learn more on the **CSS Overview** panel. -->
+Lea [este artículo](/docs/devtools/css-overview) para obtener más información sobre el panel de **Descripción General de CSS**.
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-96/index.md:40]: Same comment with inconsistent DevTools translations
```suggestion
Lea [este artículo](/docs/devtools/css-overview) para obtener más información sobre el panel de **Descripción General de CSS / Resumen de CSS**.
```

File: site/es/blog/new-in-devtools-96/index.md [hunk 3]
```diff
@@ -28,171 +28,172 @@ draft: true
...
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="CSS Overview panel", width="800", height="509" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="panel de Descripción General de CSS", width="800", height="509" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/ef26abc89035075bbdb08f1b26c1b8fd942ffc04 #}
 
 Chromium issue: [1254557](https://crbug.com/1254557)
 
 
-<!-- ## Rendering tab updates  -->
+## Actualizaciones de la pestaña de renderizado
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-96/index.md:49]: Let's add a capital R to the panel's name
```Suggestion
## Actualizaciones de la pestaña de Renderizado
```

File: site/es/blog/new-in-devtools-96/index.md [hunk 4]
```diff
@@ -28,171 +28,172 @@ draft: true
 
 {% include 'partials/devtools/es/banner.md' %}
 
-<!-- ## Preview feature: New CSS Overview panel {: #css-overview } -->
+## Función de vista previa: Nuevo panel de Descripción General de CSS {: #css-overview }
 
-<!-- Use the new **CSS Overview** panel to identify potential CSS improvements on your page.
-[Open the **CSS Overview** panel](/docs/devtools/css-overview#open), then click on **Capture overview** to generate a report of your page’s CSS. -->
+Utilice el nuevo panel **Descripción General de CSS** para identificar posibles mejoras de CSS en su página.
+[Abra el panel **Descripción general de CSS**](/docs/devtools/css-overview#open), luego haga clic en **Capture overview** para generar un informe del CSS de su página.
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-96/index.md:34]: Same comment with inconsistent DevTools translations
```suggestion
[Abra el panel **Descripción general de CSS / Resumen de CSS**](/docs/devtools/css-overview#open), luego haga clic en **Capture overview** para generar un informe del CSS de su página.
```

File: site/es/blog/new-in-devtools-96/index.md [hunk 5]
```diff
@@ -28,171 +28,172 @@ draft: true
...
+Use DevTools para emular el tema oscuro automático para ver fácilmente cómo se ve su página cuando el [Tema oscuro automático](/blog/auto-dark-theme/) de Chrome está habilitado.
 
-<!-- Chrome 96 introduces an [Origin Trial](/blog/origin-trials/) for [Auto Dark Theme](/blog/auto-dark-theme/) on Android. With this feature, the browser applies an automatically generated dark theme to light themed sites, when the user has opted into dark themes in the Operating System.  -->
+Chrome 96 presenta una [Prueba de origen](/blog/origin-trial/) para [Tema oscuro automático](/blog/auto-dark-theme/) en Android. Con esta función, el navegador aplica un tema oscuro generado automáticamente a sitios con temas claros, cuando el usuario ha optado por temas oscuros en el sistema operativo.
 
-<!-- Open the [Command Menu](/docs/devtools/command-menu/), run the **Show Rendering** command, and then set the **Emulate auto dark mode** dropdown. -->
+Abra el [Menú de comandos](/docs/devtools/command-menu/), ejecute el comando **Mostrar renderizado** y luego configure el menú desplegable **Emular modo oscuro automático**.
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-96/index.md:70]: Same comment with inconsistent DevTools translations
```suggestion
Abra el [Menú de comandos](/docs/devtools/command-menu/), ejecute el comando **Mostrar renderizado / Mostrar la renderización** y luego configure el menú desplegable **Emular modo oscuro automático**.
```

File: site/es/blog/new-in-devtools-96/index.md [hunk 6]
```diff
@@ -28,171 +28,172 @@ draft: true
...
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/b0650096c934bf60c21d51ae8a51c94e8f907d38 #}
 
 Chromium issues: [1232937](https://crbug.com/1232937), [1255586](https://crbug.com/1255586)
 
+## Panel de caché de la aplicación en desuso (deprecated) en el panel de la aplicación  {: #app-cache } -->
 
-<!-- ## Deprecated Application cache pane in the Application panel {: #app-cache } -->
-
-<!-- The [Application cache](https://developer.chrome.com/docs/devtools/storage/applicationcache/) pane in the Application panel is now removed as the support for [AppCache](https://web.dev/appcache-removal/) is removed from Chrome and other Chromium-based browsers. -->
+El panel [Caché de la aplicación](https://developer.chrome.com/docs/devtools/storage/applicationcache/) en el panel de la aplicación se eliminó como el soporte para [AppCache](https://web.dev/appcache-removal/) se elimina de Chrome y otros navegadores basados ​​en Chromium.
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-96/index.md:169]: Let's make the redaction a bit more clear:

```Suggestion
El panel [Caché de la aplicación](https://developer.chrome.com/docs/devtools/storage/applicationcache/) en el panel de Aplicación se eliminó pues el soporte para [AppCache](https://web.dev/appcache-removal/) se ha eliminado de Chrome y otros navegadores basados ​​en Chromium.
```

File: site/es/blog/new-in-devtools-96/index.md [hunk 7]
```diff
@@ -28,171 +28,172 @@ draft: true
...
-Chromium issue: [1084190](https://crbug.com/1084190) 
+Chromium issue: [1084190](https://crbug.com/1084190)
 
 
-<!-- ## [Experimental] New Reporting API pane in the Application panel {: #reporting-api } -->
+## [Experimental] Nuevo panel de la API de informes en el panel de la aplicación {: #reporting-api }
 
 {% Aside %}
-<!-- To enable the experiment, check the **Enable Reporting API panel in the Application panel** checkbox under **Settings** > **Experiments**. -->
+Para habilitar esta característica experimental, marque la casilla de verificación **Habilitar el panel de API de informes en el panel de la aplicación** que encontrará en la siguiente ruta **Configuración** > **Experimentos**.
```
Comment (and-oli) [inline site/es/blog/new-in-devtools-96/index.md:179]: I think the string with the experiment name is not translated. Ignore this comment if I'm wrong.
```suggestion
Para habilitar esta característica experimental, marque la casilla de verificación **Enable Reporting API panel in the Application panel** que encontrará en la siguiente ruta **Configuración** > **Experimentos**.
```

Comment (Caballerog) [reply site/es/blog/new-in-devtools-96/index.md:179]: You're right, experimental panel is still untranslated.

File: site/es/blog/new-in-devtools-96/index.md [hunk 8]
```diff
@@ -28,171 +28,172 @@ draft: true
...
 
-<!-- The **CSS Overview** panel is a preview feature. Our team is still actively working on it and we are looking for your [feedback](https://goo.gle/css-overview-feedback) for further enhancements. -->
+El panel **Descripción General de CSS** es una función de vista previa. Nuestro equipo todavía está trabajando activamente en él y estamos buscando sus [comentarios](https://goo.gle/css-overview-feedback) para obtener más mejoras.
 
-<!-- Read [this article](/docs/devtools/css-overview) to learn more on the **CSS Overview** panel. -->
+Lea [este artículo](/docs/devtools/css-overview) para obtener más información sobre el panel de **Descripción General de CSS / Resumen de CSS**.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="CSS Overview panel", width="800", height="509" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="panel de Descripción General de CSS", width="800", height="509" %}
```
Comment (midudev) [inline site/es/blog/new-in-devtools-96/index.md:42]: I would start with capital case.

```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="Panel de Descripción General de CSS", width="800", height="509" %}
```

File: site/es/blog/new-in-devtools-96/index.md [hunk 9]
```diff
@@ -28,171 +28,172 @@ draft: true
...
 
-<!-- You can further drill down on the information. For example, click on a color in the **Colors** section to view the list of elements that apply the same color. Click on an element to open the element in the **Elements** panel. -->
+Puede profundizar más en la información. Por ejemplo, haga clic en un color en la sección **Colores** para ver la lista de elementos que aplican el mismo color. Haga clic en un elemento para abrir el elemento en el panel **Elementos**.
 
-<!-- The **CSS Overview** panel is a preview feature. Our team is still actively working on it and we are looking for your [feedback](https://goo.gle/css-overview-feedback) for further enhancements. -->
+El panel **Descripción General de CSS** es una función de vista previa. Nuestro equipo todavía está trabajando activamente en él y estamos buscando sus [comentarios](https://goo.gle/css-overview-feedback) para obtener más mejoras.
 
-<!-- Read [this article](/docs/devtools/css-overview) to learn more on the **CSS Overview** panel. -->
+Lea [este artículo](/docs/devtools/css-overview) para obtener más información sobre el panel de **Descripción General de CSS / Resumen de CSS**.
```
Comment (midudev) [inline site/es/blog/new-in-devtools-96/index.md:40]:
```suggestion
Lea [este artículo](/docs/devtools/css-overview) para obtener más información sobre el panel de **Descripción General de CSS**.
```

¿Esto de poner dos veces lo mismo es porque en las diferentes traducciones al Español (España y Latinoamérica) tiene estas dos variantes?

File: site/es/blog/new-in-devtools-96/index.md [hunk 10]
```diff
@@ -28,171 +28,172 @@ draft: true
...
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/ca17a55104e6baf8d4ab360b484111bfa93c9b7f #}
 
 Chromium issue: [1253635](https://crbug.com/1253635)
 
 
-<!-- ## New Payload tab in the Network panel {: #payload } -->
+## Nueva pestaña de Payload en el panel Red {: #payload }
 
-<!-- Use the new **Payload** tab in the **Network** panel when you inspect a network request with payload. Previously, the payload information is available under the **Headers** tab. -->
+Utilice la nueva pestaña **Payload** en el panel **Red** cuando inspeccione una solicitud de red con payload. Anteriormente, la información del payload estaba disponible en la pestaña **Encabezados**.
```
Comment (midudev) [inline site/es/blog/new-in-devtools-96/index.md:96]: La traducción en Chrome de Payload es Carga útil.

```suggestion
Utilice la nueva pestaña **Carga útil** en el panel **Red** cuando inspeccione una solicitud de red con payload. Anteriormente, la información del payload estaba disponible en la pestaña **Encabezados**.
```

**Review by midudev — APPROVED** [review]

File: site/es/blog/new-in-devtools-96/index.md (hunk 9 above)
Comment (Caballerog) [reply site/es/blog/new-in-devtools-96/index.md:40]: Sí, es esa la razón, puesto que depende del idioma que se tenga en el navegador la traducción que aparece es una u otra.

**Review by chybie — APPROVED** [review]

**Comment by chybie** [comment]
@googlebot I fixed it.

**Comment by chybie** [comment]
@Caballerog and @midudev can you please type`@googlebot I fixed it.` in the comment? I'm not able to merge the PR without that.

The CLA process changed. Thanks!

**Comment by Caballerog** [comment]
@googlebot I fixed it.

**Pull request merged**